.travis.yml

```diff
@@ -46,8 +46,9 @@ install:
       --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
       --name centos-${OS_VERSION} local/centos:${OS_VERSION} init
   - |
-    sudo docker exec centos-${OS_VERSION} yum -y install \
-      https://repo.saltstack.com/yum/redhat/salt-repo-latest-2.el${OS_VERSION}.noarch.rpm
+    sudo docker exec centos-${OS_VERSION} rpm --import https://repo.saltproject.io/py3/redhat/${OS_VERSION}/x86_64/${SALT_VERSION}/SALTSTACK-GPG-KEY.pub
```

Comment:
Is this needed?

Reply:
Was following the instructions provided here https://repo.saltproject.io/#rhel